src/os/bluestore/BlueStore.cc

```diff
@@ -4195,6 +4202,23 @@ void BlueStore::_set_blob_size()
 
 int BlueStore::_set_cache_sizes()
 {
+  // set osd_memory_target *default* based on cgroup limit?
+  // (do this before we fetch the osd_memory_target value!)
+  double cgroup_ratio = cct->_conf.get_val<double>(
+    "osd_memory_target_cgroup_limit_ratio");
+  if (cgroup_ratio) {
```

Comment:
Converting a `double` to `bool` makes me squeamish. How about `cgroup_ratio > 0.0`?